documentation/asciidoc/computers/config_txt/what_is_config_txt.adoc

```diff
@@ -44,7 +44,21 @@ Causes the content of the specified file to be inserted into the current file.
 
 For example, adding the line `include extraconfig.txt` to `config.txt` will include the content of `extraconfig.txt` file in the `config.txt` file.
 
-*Include directives are not supported by bootcode.bin or the EEPROM bootloader*
+[NOTE]
+====
+
+*Include directives are not supported by the bootcode.bin or EEPROM bootloaders*.
+
+Settings which are handled by the bootloader and so which will only take effect if they are specified in `config.txt` (rather than any additional included file) are:
```

Comment:
I'd be happy to keep this bit where you document which settings don't work when used in an included file, as I suspect only a _very_ small proportion of users actually use config.txt includes.

Reply:
I think a small note block about include files is okay. NOTES per individual items, not so much.

Reply:
> I think a small note block about include files is okay. NOTES per individual items, not so much.

https://github.com/raspberrypi/documentation/commit/d6900bb65b53e8bb56e11bc8d64080721b3f2e2e removes the individual NOTEs.